info.yml

```diff
@@ -76,6 +76,14 @@ configuration:
                      setting is an empty string, no logo will be applied.
         default_value: ""
 
+    hook_reviewsubmission_quicktime_settings:
+        type: hook
+        description: "Called when generating the default settings for creating
+                     a Quicktime. The return value is a dictionary where the
+                     keys are knob names for the write node, and the values are
+                     the corresponding node values."
+        default_value: reviewsubmission_quicktime_settings
```

Comment:
Use the new style default value for hooks ( see docs here: https://support.shotgunsoftware.com/entries/95442678#The%20hook%20data%20type )

Suggested default: `{self}/quicktime_settings.py`.